src/main/java/unbanner/Course.java

```diff
@@ -0,0 +1,101 @@
+package unbanner;
+
+import org.springframework.data.annotation.Id;
+
+public class Course implements Storable{
+
+
+    String department;
+    int crn; //course number
+    int credits;
+    String description;
+    String learningObjectives;
+    int prerequisiteCreditLevel; //senior, graduate, etc.. ENUM these in global
```

Comment:
if you want this to be an Enumerated type (good idea) we should declare it as such (not as an int). This will likely have to be defined as its own class and imported.